src/Fmt_ast.ml

```diff
@@ -1866,6 +1866,29 @@ and fmt_expression c ?(box = true) ?pro ?epi ?eol ?parens ?(indent_wrap = 0)
             $ fmt "@;<1000 0>"
             $ fmt_expression c (sub_exp ~ctx exp) )
         $ fmt_atrs )
+  | Pexp_letmodule (name, pmod, exp) when Poly.(c.conf.let_module = `Sparse)
```

Comment:
I don't like the code duplication. I think it would be better to clean up the previous code to allow this feature instead.